app/assets/stylesheets/markdown-editor.scss

```diff
@@ -0,0 +1,142 @@
+.md-footer,
+.md-header {
+  background: $sidebars-background;
+  border: 0;
+  display: block;
+  height: 42px;
+  margin: 0;
+  padding: 6px 6px 0;
+
+  [class^="entypo-"],
+  [class*="entypo-"],
+  .glyphicon {
+    color: $black;
+  }
+}
+
+.md-header,
+.nav-tabs {
+  border-bottom: 0;
+  margin: 0;
+  z-index: 1;
+}
+
+.md-header.btn-toolbar {
+  background-color: $background-grey;
+  overflow: hidden;
+
+  .btn-group { margin-bottom: 8px; }
+
+  @media(max-width: $screen-xs) {
+    [data-handler="bootstrap-markdown-cmdList"],
+    [data-handler="bootstrap-markdown-cmdListO"] {
+      display: none;
+    }
+
+    [data-handler="bootstrap-markdown-cmdCode"] {
+      // !important is needed to override BS' specific rules
+      // scss-lint:disable ImportantRule
+      border-bottom-left-radius: $border-radius-small !important;
+      border-top-left-radius: $border-radius-small !important;
+      // scss-lint:enable ImportantRule
+    }
+  }
+
+  [class^="entypo-"],
+  [class*="entypo-"] {
+    font-size: 13px;
+  }
+
+  [data-handler="bootstrap-markdown-cmdUrl"],
+  [data-handler="bootstrap-markdown-cmdImage"],
+  [data-handler="bootstrap-markdown-cmdList"],
+  [data-handler="bootstrap-markdown-cmdListO"],
+  [data-handler="bootstrap-markdown-cmdCode"],
+  [data-handler="bootstrap-markdown-cmdQuote"] {
+    height: 28.5px;
+    line-height: 1.25;
+    width: 33px;
+  }
+}
+
+.md-cancel {
+  box-sizing: content-box;
+
+  &,
+  .entypo-cross {
+    color: $text-grey;
+    font-size: 18px;
+    height: 18px;
+    line-height: 18px;
+    width: 18px;
+  }
+
+  &:hover .entypo-cross { color: $text; }
+}
+
+
+.md-preview {
+  background: $white;
+  color: $text-color;
+  // !important is needed to override the CSS rules dynamically added to the element
+  // scss-lint:disable ImportantRule
+  height: auto !important;
+  // scss-lint:enable ImportantRule
+  min-height: 90px;
+  overflow: auto;
+  position: relative;
+  // !important is needed to override the CSS rules dynamically added to the element
+  // scss-lint:disable ImportantRule
+  width: 100% !important;
+  // scss-lint:enable ImportantRule
+  z-index: 10;
+}
+
+.md-controls {
+  float: right;
+  padding: 3px;
+
+  .md-control {
+    color: $text-grey;
+    padding: 3px;
+    padding-left: 10px;
+    right: 5px;
+  }
+}
+
+
+.write-preview-tabs {
+  &,
+  & .full-height {
+    height: 36px;
+  }
+
+  > li {
+    > a { padding: 7px 15px; }
+
+    &:not(.active) * { color: $brand-primary; }
+
+    &.active * { color: $black; }
+  }
+
+  a:focus { outline: none; }
+
+  li {
+    &:not(.active) a:focus,
+    &:not(.active) a:hover {
+      background-color: transparent;
+      border: 1px solid transparent;
+    }
+
+    &.active:focus { color: $black; }
+  }
+
+  .tab-help-text { margin-left: 5px; }
```

Comment:
This isn't needed anymore since icons and text will never be visible at the same time. Having the margin there means that the text won't be centered.